### PR TITLE
WIP: perfect-numbers

### PR DESCRIPTION
**Pull request by sbromberger**
ping @vaeng for guidance and feedback. I ran `configlet` for the docs and for `config.json` formatting, and added a UUID and other metadata to the track-level `config.json`.

### CODE REVIEW

File: exercises/practice/perfect-numbers/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "authors": [
+    "sbromberger"
+  ],
+  "contributors": [
```
Comment (vaeng) [inline exercises/practice/perfect-numbers/.meta/config.json:5]: I'm pretty sure it is only you as an author, and you mention the version you forked from to hint that others has worked on the exercise as well.
https://exercism.org/docs/building/tracks/practice-exercises:
>- contributors: The GitHub username(s) of the exercise approach introduction's contributor(s) (optional)
>-  Including reviewers if their reviews are meaningful/actionable/actioned.

File: exercises/practice/perfect-numbers/.meta/example.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#include <cmath>
+#include <numeric>
```
Comment (vaeng) [inline exercises/practice/perfect-numbers/.meta/example.cpp:2]: What's that for?

Comment (sbromberger) [reply exercises/practice/perfect-numbers/.meta/example.cpp:2]: oops. I don't need it anymore.

Is there a tool that will warn of unnecessary `#include`s?

File: exercises/practice/perfect-numbers/.meta/config.json (hunk 1 above)
Comment (sbromberger) [reply exercises/practice/perfect-numbers/.meta/config.json:5]: Where do I put the credit for the version I started with (golang)?

File: exercises/practice/perfect-numbers/.meta/example.cpp (hunk 2 above)
Comment (vaeng) [reply exercises/practice/perfect-numbers/.meta/example.cpp:2]: You could use on of these: 
https://stackoverflow.com/questions/614794/detecting-superfluous-includes-in-c-c

Comment (vaeng) [inline exercises/practice/perfect-numbers/.meta/example.cpp:2]:
```suggestion
```

File: exercises/practice/perfect-numbers/.meta/config.json (hunk 1 above)
Comment (vaeng) [reply exercises/practice/perfect-numbers/.meta/config.json:5]: I mixed up concept and practice exercises. I think it should be possible in both.

So I would suggest to keep "ferhatelmas" as a contributor, but drop the rest.

File: exercises/practice/perfect-numbers/.meta/example.cpp (hunk 2 above)
Comment (sbromberger) [reply exercises/practice/perfect-numbers/.meta/example.cpp]: Thanks!

File: exercises/practice/perfect-numbers/.meta/config.json [hunk 3]
```diff
@@ -3,20 +3,7 @@
     "sbromberger"
   ],
   "contributors": [
-    "alebaffa",
-    "bitfield",
-    "ekingery",
     "ferhatelmas",
```
Comment (vaeng) [inline exercises/practice/perfect-numbers/.meta/config.json:6]:
```suggestion
    "ferhatelmas"
```
I think you have to remove that , to make please json.

**Review by vaeng — CHANGES_REQUESTED** [review]

**Comment by sbromberger** [comment]
@vaeng thanks for all the guidance on this! Looking forward to creating more.

**Pull request merged**